Firestore/Example/Podfile

```diff
@@ -8,6 +8,7 @@ target 'Firestore_Example' do
   target 'Firestore_Tests' do
     inherit! :search_paths
     pod 'OCMock'
+    pod 'GoogleTest', :podspec => 'Tests/GoogleTest/GoogleTest.podspec'
```

Comment:
This patch set order looks a little odd.  It looks like you're referencing the GoogleTest.podspec before it's present in the patch series.

Reply:
This is really weird. Locally the order is correct; github is displaying them in the wrong order.

git log --reverse wilhuff/style..wilhuff/googletest 
commit 50490fd74c3bdf61769f6af1766b02c68a881283
Author: Marek Gilbert <mcg@google.com>
Date:   Wed Nov 1 06:27:08 2017 -0700

    Initial, minimal GoogleTest.podspec
    
    Does not yet pass pod spec lint

commit f72072df2eb84e076a4e7448316f31d02f832429
Author: Marek Gilbert <mcg@google.com>
Date:   Wed Nov 1 06:32:11 2017 -0700

    Add pod-local xcconfig to enable the build to find its own headers

commit 1589a63f85eeaa07e98d36d1eb242a09681b7f29
Author: Marek Gilbert <mcg@google.com>
Date:   Wed Nov 1 06:36:53 2017 -0700

    Prevent duplicate symbols by excluding gtest-all.cc

commit f1e53b0d2d6202b95b5f15f61e976e41e2557fb5
Author: Marek Gilbert <mcg@google.com>
Date:   Fri Oct 27 08:35:50 2017 -0700

    Use GoogleTest as a dependency of Firestore_Tests

commit 5561b32001a7693f8d19f042f7ae4db3bcc7e66c
Author: Marek Gilbert <mcg@google.com>
Date:   Wed Nov 1 06:50:54 2017 -0700

    Exclude gtest/internal/custom headers to prevent analysis warnings

commit 44f93fbd65df4298062c051f73c79c5dca5c8db6
Author: Marek Gilbert <mcg@google.com>
Date:   Wed Nov 1 06:55:18 2017 -0700

    Unflatten gtest includes to make public headers work

commit 57ce9a62625eb4f9c9f26b34dbea46d9647c5a83
Author: Marek Gilbert <mcg@google.com>
Date:   Fri Oct 27 08:39:37 2017 -0700

    Remove top-level leveldb-library from HEADER_SEARCH_PATHS
    
    This path is only needed to build leveldb-library, not to get its public
    headers

commit be9d39bee3421a7d98d599bbedd56b3cb54924b7
Author: Marek Gilbert <mcg@google.com>
Date:   Wed Nov 1 07:08:16 2017 -0700

    Add string_util_test to the project and get it to build

commit a639c9554ce31aa4bd09e54779b83dd8d0b846ce (origin/wilhuff/googletest, wilhuff/googletest, autoid)
Author: Marek Gilbert <mcg@google.com>
Date:   Fri Oct 27 12:04:43 2017 -0700

    Implement FSTGoogleTestTests, a bridge between GoogleTest and XCTest